plot_trunc_filter.py

```diff
@@ -0,0 +1,37 @@
+import matplotlib.pyplot as plt
+import numcodecs
+import zarr
+import tifffile
+import numpy as np
+compress_zarr = __import__("compress-zarr")
```

Comment:
Is this to avoid a circular dependency?  I would just move `eval_quality` in here if so.

Reply:
Oh, it's because of the hyphen.  Go ahead and rename `compress-zarr.py` to `compress_zarr.py`.  You could do the same for `compress-h5.py`